Sources/SwiftCurrent_SwiftUI/Views/WorkflowItem.swift

```diff
@@ -49,7 +49,9 @@ public struct WorkflowItem<F: FlowRepresentable & View, Wrapped: View, Content:
         ViewBuilder {
             if launchStyle == .navigationLink, let content = content {
                 content.navLink(to: nextView, isActive: $isActive)
-            } else if let body = model.body?.extractErasedView() as? Content, elementRef == nil || elementRef === model.body, launchStyle == .default {
+            } else if (wrapped as? WorkflowItemPresentable)?.workflowLaunchStyle == .modal, let content = content {
+                content.testableSheet(isPresented: $isActive) { nextView }
```

Comment:
There might be a better name for `$isActive` but I'm not sure what it is yet.  When I first read this, it always takes me a minute to realize what `isActive` is referring to (the next screen being active/this link being active, NOT the current screen being active).

Reply:
Hrm, that's what navigation links call it, modal call it `isPresented`. Both of those would cause your confusion. Then again with this highly recursive structure I'm not sure there's a very clear way to describe it.

Reply:
What about: `launchStyleIsActive`?  Or `launchStyleModifierIsActivated`?

Probably should go with: `isThisEverGoingToEndRichard`

Reply:
I'll +1 `launchStyleIsActive`

Reply:
`launchStyleIsActive` is deceptive because the launch style is not the active thing. `launchStyleModifierIsActivated` is certainly verbose, but a poor descriptor as NavLinks are not modifiers, even if you're being incredibly pedantic about how we made a thing that *looks* kind of like a modifier for nav links it's not actually a modifier, it's just an extension function. 

`isThisEverGoingToEndRichard` probably doesn't give maintainers the opportunity they need to understand any better than `isActive` does.

Reply:
I disagree. `isThisEverGoingToEndRichard` clearly points to who caused the problem and maintenance nightmare.